package.json

```diff
@@ -32,7 +32,7 @@
     "eslint-fix": "npm run eslint -- --fix",
     "prettier": "prettier . --check",
     "prettier-write": "prettier . --write",
-    "preview:api": "NODE_ENV=production npm run build && node bin/rest-on-couch-server.js",
+    "preview:api": "NODE_ENV=production npm run build && node --env-file=.env.dev bin/rest-on-couch-server.js",
```

Comment:
Can you commit the `.env.dev`? if not, maybe document how it should be in the readme.

Reply:
Actually it was committed: https://github.com/cheminfo/rest-on-couch/blob/main/.env.dev